cmd/rpcfuzz/rpcfuzz.go

```diff
@@ -1672,7 +1730,7 @@ func ArgsTransactionHash(ctx context.Context, rpcClient *rpc.Client, tx *RPCTest
 	return func() []interface{} {
 		resultHash, _, err := prepareAndSendTransaction(ctx, rpcClient, tx)
 		if err != nil {
-			log.Fatal().Err(err).Msg("Unable to execute transaction")
+			log.Error().Err(err).Msg("Unable to execute transaction")
```

Comment:
Interesting change - was the use of `Fatal` a mistake before or is there a reason why we want to continue running now when previously we did not.

Reply:
I felt like exiting the conformance test midway is unnecessary since the conformance test will report it as a failed test.